js/filter.js

```diff
@@ -0,0 +1,60 @@
+const mapFilter = document.querySelector('.map__filters');
+const typeFilter = mapFilter.querySelector('#housing-type');
+const priceFilter = mapFilter.querySelector('#housing-price');
+const roomsFilter = mapFilter.querySelector('#housing-rooms');
+const guestsFilter = mapFilter.querySelector('#housing-guests');
+const featuresFilter = mapFilter.querySelector('#housing-features');
+
+const PRICES = {
+  'low': 10000,
+  'high': 50000,
+  'any': 0,
+}
+
+// Фильтрация по цене
+const getPriceFilter = (data) => {
+  switch (priceFilter.value) {
+    case 'low':
+      return data.offer.price < PRICES[priceFilter.value];
+    case 'middle':
+      return (data.offer.price >= PRICES['low']) && (data.offer.price <= PRICES['high']);
+    case 'high':
+      return data.offer.price > PRICES[priceFilter.value];
+    default:
+      return PRICES;
```

Comment:
нужно возвращать по дефолту значение такого же типа
```suggestion
      return false;
```